src/content/popup/popup.css

```
@@ -1250,6 +1250,10 @@
   .meta.era .seireki {
     font-size: calc(22 / 14 * var(--base-font-size));
   }
+  .meta.era .secondary {
+    font-size: calc(14 / 14 * var(--base-font-size));
```

I guess this could just be:

```suggestion
    font-size: var(--base-font-size);
```

---

Definitely. How did I not see that? 😄